src/runtime_src/core/common/info_memory.cpp

```diff
@@ -506,34 +506,14 @@ get_ps_kernels(const xrt_core::device *device)
 }
 
 ptree_type
-populate_cus(const xrt_core::device *device)
+populate_cus(const xrt_core::device* device, const std::vector<xq::kds_cu_info::data_type>& cu_stats, const std::vector<xq::kds_scu_info::data_type>& scu_stats)
 {
   scheduler_update_stat(device);
```

Comment:
This call is no longer required as this is being called in populating hw context information.
All other changes are looking good.

Reply:
Good catch. Thank you.